pkgs/by-name/fi/finalmouse-udev-rules/package.nix

```diff
@@ -0,0 +1,37 @@
+{
+  fetchFromGitHub,
+  lib,
+  stdenv,
+}:
+
+stdenv.mkDerivation {
+  pname = "finalmouse-udev-rules";
+  version = "0-unstable-2025-05-05";
+
+  src = fetchFromGitHub {
+    owner = "teamfinalmouse";
+    repo = "xpanel-linux-permissions";
+    rev = "60c4ed794bd946e467559cc572cf25bb99bf04b6";
+    hash = "sha256-E43jZwAgkfwbyylmemD+1n+ykDdv1q8TSp4vptUFteE=";
```

Comment:
```suggestion
    hash = "sha256-E2xhm+8fFlxgIKjZlAvosLk/KgbmLk01BjK++y8laBc=";
```

Reply:
Smh that's so stupid, I even checked before updating my PR. But the build didn't fail, did I need to do a flake update? Anyways I updated the hash in the commit below.

Reply:
It's very probably that you had the a cached version build with that hash that was automatically reused, failling silently